src/components/message-input/index.tsx

```diff
@@ -121,6 +129,17 @@ export class MessageInput extends React.Component<Properties, State> {
         markup={userMentionsConfig.markup}
         displayTransform={userMentionsConfig.displayTransform}
       />,
+      <Mention
```

Comment:
Does this need to be here? When we switched from saving the colon based token to just the utf character I thought we didn't need this any more.

Reply:
https://github.com/m3m3n70/zero-web/blob/development/src/components/mentions-text-area/index.tsx#L84-L88

Reply:
when I remove it and write start typing `:+1:` it doesn't convert to emoji